apps/teams-test-app/src/components/Custom.tsx

```diff
@@ -30,7 +42,8 @@ const RegisterCustomHandler = (): React.ReactElement =>
 
 const CustomAPIs: React.FC = () => (
   <ModuleWrapper title="Custom">
-    <CustomApi />
```

Comment:
I believe that your test failure is because there's a test looking for a component that was getting constructed inside of this component called #box_customAPI:

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/18446904/4e110ae8-4ecd-49c7-958b-8ff84e0ce5f6)

This probably means you still have a test in the json that is looking for this that you need to update. If you have questions about how to fix this, please post them in our e2e test channel (or reply here)

Reply:
I believe this was due to the version check not being updated in hubSDK. Looks like after that went in the gate started passing.